README.md

```diff
@@ -1,30 +1,59 @@
-# Pyodide
+**[What is Pyodide?](#what-is-pyodide)** |
+**[Try Pyodide](#try-pyodide-no-installation-needed)** |
+**[Getting Started](#getting-started)** |
+**[Contributing](#contributing)** |
+**[License](#license)** |
+**[Help and Resources](#help-and-resources)**
+
+# [Pyodide](https://github.com/iodide-project/pyodide)
 
 
 [![Build Status](https://circleci.com/gh/iodide-project/pyodide.png)](https://circleci.com/gh/iodide-project/pyodide)
 
 The Python scientific stack, compiled to WebAssembly.
 
-It provides transparent conversion of objects between Javascript and Python.
-When inside a browser, this means Python has full access to the Web APIs.
+[**Try Pyodide and Iodide in your browser**](https://alpha.iodide.io/notebooks/300/)
+
+## What is Pyodide?
+
+**Pyodide** brings the Python runtime to the browser via WebAssembly, along with the Python scientific stack including NumPy, Pandas, Matplotlib, parts of SciPy, and NetworkX.
 
-**While closely related to the [iodide project](https://iodide.io), Pyodide may
-be used standalone in any context where you want to run Python inside a web
-browser.**
+**Pyodide** provides transparent conversion of objects between Javascript and Python.
+When used inside a browser, Python has full access to the Web APIs.
 
-For more information, see [the demo](https://alpha.iodide.io/notebooks/300/) and the
+While closely related to the [iodide project](https://iodide.io), a tool for *iterate scientific computing and communication for the web*, Pyodide goes beyond running in a notebook environment. To maximize the flexibility of the modern web, **Pyodide** may
+be used standalone in any context where you want to **run Python inside a web
+browser**.
+
+## Try Pyodide (no installation needed)
+
+For more information, try [the demo](https://alpha.iodide.io/notebooks/300/) and look through the
 [documentation](https://github.com/iodide-project/pyodide/tree/master/docs).
 
-# Downloading pre-built versions
+## Getting Started
+
+Pyodide offers three different ways to get started depending on your needs and technical resources.
+These include:
 
-Pre-built versions of Pyodide are available
-[here](https://github.com/iodide-project/pyodide/releases/).
+- [Download a pre-built version](#download-a-pre-built-version) (the quickest way to get started)
+- [Build Pyodide from source](#building-from-source) (this method requires installing prerequistes and using `make`. Primarily for Linux users who want to experiment or contribute back to the project.)
+- [Use a Docker image](#using-docker) (recommended for Windows and macOS users and for Linux users who prefer a Debian-based Docker image on Docker Hub with the dependencies
+already installed)
 
-# Building
+
+### Download a pre-built version
+
+Pre-built versions of Pyodide may be downloaded from
+this repos's [releases page](https://github.com/iodide-project/pyodide/releases/).
```

Comment:
Nit: Maybe say "repository's" (just felt a little weird to abbreviate with the `'s`)...

Reply:
Makes sense.